.github/workflows/e2e-suite.yml

```diff
@@ -28,6 +32,15 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event_name == 'workflow_dispatch' && inputs.sha != '' || github.event_name == 'push' || github.event_name == 'pull_request'
     steps:
+      - name: Validate Test Path
```

Comment:
I think you might have merged older local version of `dev` branch

Reply:
yes , raised the new PR